test/state/state.cpp

```diff
@@ -460,6 +480,88 @@ TransactionReceipt transition(const StateView& state_view, const BlockInfo& bloc
     assert(sender_acc.nonce < Account::NonceMax);  // Required for valid tx.
     ++sender_acc.nonce;                            // Bump sender nonce.
 
+    auto delegation_refund = 0;
+
+    for (const auto& auth : tx.authorization_list)
+    {
+        // 1. Verify the chain id is either 0 or the chain’s current ID.
+        // FIXME: Transaction chain id is not the chain id. Can it be 0?
```

Comment:
Please check if tx.chain_id may be 0.

Reply:
I checked this and `tx.chain_id` potentially can be 0 but it has no special meaning. So if we get a tx with chain_id 0 the authorization will be valid iff the authorisation chain_id is 0 so this is kinda fine.

On the other hand, a tx with chain_id 0 should not reach this point because by EIP-155 the configured chain ID must not be 0. We don't really have a check for this because it looks like state tests don't explore this area.